tests/ui/wrong_self_convention.stderr

```diff
@@ -191,5 +175,21 @@ LL |         fn from_i32(self);
    |
    = help: consider choosing a less ambiguous name
 
+error: methods with the following characteristics: (`to_*` and `self type is  Copy`) usually take self by value
```

Comment:
Looking once more at the resulting error message, this gave me pause: "\`self type is  Copy\`" should probably read "\`self\` type is \`Copy\`" (note the double space and the backticks).

Reply:
Good catch, thanks!